common/components/Header/components/TabsOptions.jsx

```diff
@@ -23,7 +23,8 @@ const tabs = [
         link: 'view-wallet'
     },
     {
-        name: 'Help'
+        name: 'Help',
```

Comment:
ps: When I comment like this, take it or leave it. I'll make it clear when I feel strongly about something existing in a particular way. For most things, I'm going to share insightsabout what we've learned and you can use those as you proceed. Please don't make a decision based solely on something I've said, especially if it compromises something else.